python/enthic/app.py

```diff
@@ -178,23 +172,73 @@ def pre_cast_integer(probe):
     return str(probe) if probe.__class__ is int else probe if probe.isnumeric() is True else None
 
 
-def result_array(probe, limit, offset=0):
+def ape_list(ape_code):
+    """
+    APE code are not save in the original format.
+    This function returns the corresponding code in MySQL or None
+    if the code does not exist
+
+        :param ape_code : the list of APE for he filter
+        :param APE_CODE : all known APE code
```

Comment:
APE_CODE  is not a parameter. Explain above why and how you are using it.

Reply:
it is due to the fact that at first, I had taken APE_CODE as a parameter but afterwards I said to myself that it was no longer necessary because this variable was already available in the app.py module.
I've forgotten to remove it in the doc-string